partiql-planner/src/testFixtures/resources/catalogs/default/pql/t_item.ion

```diff
@@ -10,7 +10,7 @@
     },
     {
       name: "t_bool_nul",
-      type: ["bool","null"],
+      type: "bool",
```

Comment:
Yeah, I thought to do that, but I opted to avoid doing that due to the diff on tests. I can either include that change or follow up with a clean up of the duplicate tests.